pom.xml

```diff
@@ -68,6 +68,11 @@
             <artifactId>display-url-api</artifactId>
             <version>0.2</version>
         </dependency>
+        <dependency>
```

Comment:
Oh I am not happy about this being escalated from test. Ping me tomorrow. We need to find a different way

Reply:
I first tried with `SCMSource.SourceByItem.findSource(item)` but the problem there was that it only found the source for actual branch jobs, so it doesn't work on `afterSave` because there are no branch jobs available at that time, so still no repository to discover.

Reply:
```
if (item instanceof SCMSourceOwner) {
 ...
}
```